libpysal/graph/tests/test_builders.py

```diff
@@ -194,6 +198,38 @@ def setup_method(self):
         )
         self.gdf_str = self.gdf.set_index("placeid")
 
+    def test_kernel_precompute(self):
```

Comment:
```suggestion
    def test_kernel_precompute(self):
        sklearn = pytest.importorskip("sklearn")
```
Import via importorskip

Reply:
i did not know about importorskip. thats awfully handy